onnxruntime/core/framework/kernel_registry.cc

```diff
@@ -233,9 +233,10 @@ Status KernelRegistry::TryCreateKernel(const onnxruntime::Node& node,
                                        const FuncManager& funcs_mgr,
                                        const DataTransferManager& data_transfer_mgr,
                                        /*out*/ std::unique_ptr<OpKernel>& op_kernel) const {
-  const KernelCreateInfo* kernel_create_info;
+  const KernelCreateInfo* kernel_create_info = nullptr;
   ORT_RETURN_IF_ERROR(TryFindKernel(node, execution_provider.Type(), &kernel_create_info));
-
+  ORT_ENFORCE(nullptr != kernel_create_info, "kernel info must not be NULL");
```

Comment:
was xcode requiring the throw or just the nullptr init? i'll trade a throw in MLAS for this one ;)

Reply:
For this particular case, it requires check if "kernel_create_info" is NULL.